SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2114,6 +2116,14 @@ bool USpatialNetDriver::FindAndDispatchStartupOpsServer(const TArray<Worker_OpLi
 {
 	TArray<Worker_Op*> FoundOps;
 
+	Worker_Op* EntityQueryReservationResponseOp = nullptr;
+	FindFirstOpOfType(InOpLists, WORKER_OP_TYPE_ENTITY_QUERY_RESPONSE, &EntityQueryReservationResponseOp);
```

Comment:
The name of this query probably shouldn't have "Reservation" in it (assuming this was copy-pasted from the EntityIdReservation stuff below)

Reply:
Updated.